legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectApiTestResource.java

```diff
@@ -86,16 +81,17 @@ public void testGetProject() throws LegendSDLCServerException
         assertEquals(Sets.mutable.withAll(tags), Sets.mutable.withAll(retrievedProject.getTags()));
     }
 
-    @Test
-    public void testUpdateProject()
+    public void runUpdateProjectTest()
     {
         String projectName = "TestProjectThree";
         String description = "A test project.";
-        ProjectType projectType = ProjectType.PROTOTYPE;
+        ProjectType projectType = ProjectType.PRODUCTION;
         String groupId = "org.finos.sdlc.test";
         String artifactId = "testprojthree";
         List<String> tags = Lists.mutable.with("doe", "moffitt");
 
+        System.out.println("gitlabProjectApi: " + gitLabProjectApi.toString());
```

Comment:
Please remove this System.out.println call.

Reply:
Fixed!